### DISCUSSION TITLE
Could android xml (from SVG) be used instead of png ?

### DISCUSSION THREAD

**Discussion by doothez** [zoontek/react-native-bootsplash#586] [Q&A] [answered]
I used Android xml (from SVG) file with v3.2.4, but the same way is not working well with v5.4.1.
Is that not supported anymore ?

This is the way I used Android xml with v3.2.4, if it is still supported, can you guide how to do with v5.4.1 ?

**android/app/src/main/res/values/styles.xml**
```
<resources>

    <!-- Base application theme. -->
    <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
        <!-- Customize your theme here. -->
        <item name="android:textColor">#000000</item>
        <item name="android:editTextBackground">@drawable/rn_edit_text_material</item>
    </style>

    <style name="BootTheme" parent="AppTheme">
        <item name="android:background">@drawable/launchscreen</item>
    </style>

</resources>
```
**android/app/src/main/res/drawable/launchscreen.xml**
```
<?xml version="1.0" encoding="utf-8"?>

<layer-list xmlns:android="http://schemas.android.com/apk/res/android" android:opacity="opaque">
    <item android:drawable="@color/launchscreen_background" />
    <item android:drawable="@drawable/ic_splashlogo"  android:gravity="center" android:bottom="113dp" android:width="227dp" android:height="52dp" />
</layer-list>
```

**Comment by zoontek** [comment]
You can use vectors, but not full screen ones, it has to respect a certain size and position:

https://developer.android.com/develop/ui/views/launch/splash-screen

This is mandatory since Android 12+.

**Reply by doothez** [reply]
Thanks for the answer, but I think it is not using` react-native-bootsplash`, right ?

**Reply by zoontek** [reply]
@doothez This library relies on standards (the SplashScreen API) 🙂

**Reply by doothez** [reply]
Oh that's good news to me.
But since I'm not good at `react-native-bootsplash` and using the `SplashScreen API` together,
Can you provide a more detailed guide on how to display vector files in bootSplash using `react-native-bootsplash` and `SplashScreen API` ?

**Reply by zoontek** [reply, marked as answer]
Create vector drawables for bootSplashLogo, bootSplashBrand, replace the pngs, done 😄

**Reply by doothez** [reply]
OK, thanks for the answer
